pkg/microservice/aslan/core/service/service/service.go

```diff
@@ -349,6 +360,150 @@ func CreateK8sWorkLoads(ctx context.Context, requestID, username string, product
 	return commonrepo.NewWorkLoadsStatColl().UpdateWorkloads(workLoadStat)
 }
 
+type ServiceWorkloads struct {
+	EnvName     string `bson:"env_name"         json:"env_name"`
+	Name        string `bson:"name"             json:"name"`
+	Type        string `bson:"type"             json:"type"`
+	ProductName string `bson:"product_name"     json:"product_name"`
+	Operation   string `bson:"operation"        json:"operation"`
```

Comment:
Operation is not part of a workload definition, it is strange to add it together with other fields